cocos/base/CCProperties.cpp

```diff
@@ -480,10 +480,8 @@ char* Properties::trimWhiteSpace(char *str)
         return str;
     }
 
-    char *end;
-
     // Trim leading space.
-    while (isspace(*str))
+    while (*str != '\0' && isspace(*str))
```

Comment:
Xcode reports this warning:
![screen shot 2017-03-13 at 11 28 26](https://cloud.githubusercontent.com/assets/493372/23840455/41b9a4b2-07e0-11e7-91fd-b340627629ca.png)